ProjectSidewalk/SidewalkWebpage
# 2640 redesigned smiley scale

Resolves #2640 

This is a brief description of the problem solved or feature implemented and how you implemented it.

The smileys used on gallery, validate, and audit page started from a 'smiling face' which needed a redesign to start from a 'neutral face'. Additionally the icons were not consistent and used PNGs instead of SVGs—both of these issues are addressed.

##### Before/After screenshots (if applicable)
The smiley redesign on the filter section of the gallery page.
Before
<img width="619" alt="Screenshot 2022-12-18 at 9 17 52 AM" src="https://user-images.githubusercontent.com/8168506/208310826-ea56ec62-858d-408c-9443-073f587014cc.png">

After
![image](https://user-images.githubusercontent.com/8168506/208310208-4c8556cf-7fc8-4dc2-b4c1-6ec98d9a8db7.png)

More screenshots here: https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2640#issuecomment-1356744975

##### Testing instructions
1. Ensure that all the smileys on gallery, validate (both desktop and mobile), and audit pages are updated and consistent.
2. Ensure that all the smiley interactions on the above pages work as expected. No change in behavior is expected. 

##### Things to check before submitting the PR
